init-service/src/test/java/uk/ac/ox/ndph/mts/init_service/service/PractitionerServiceTest.java

```diff
@@ -7,18 +7,24 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.web.reactive.function.client.WebClient;
 import uk.ac.ox.ndph.mts.init_service.exception.DependentServiceException;
+import uk.ac.ox.ndph.mts.init_service.model.IDResponse;
 import uk.ac.ox.ndph.mts.init_service.model.Practitioner;
+import uk.ac.ox.ndph.mts.init_service.model.RoleAssignment;
 
 import java.io.IOException;
 import java.util.Collections;
 import java.util.List;
 
 import static org.assertj.core.api.Assertions.fail;
```

Comment:
Probably don't want to mix your unit test libraries here - use org.junit.jupiter.api.Assertions#fail() and #assertNotNull - won't change meaning of test

Reply:
Thanks, good spot, I've changed this.